f5_openstack_agent/lbaasv2/drivers/bigip/test/test_network_service.py

```diff
@@ -235,3 +440,23 @@ def test_assign_route_domain(self, network_service, network, subnet):
             network['provider:segmentation_id'] = 604
             network['provider:network_type'] = ''
             network_service.assign_route_domain(tenant_id, network, subnet)
+
+    def test_get_subnets_to_assure(self, network_service, service):
+        subnets = network_service._get_subnets_to_assure(service)
+        assert len(subnets) == 3
```

Comment:
Can we also assure that the subnets dict has the proper values for network, subnet, and is_for_member?